contracts/WildcardsQV.sol

```diff
@@ -41,83 +61,113 @@ contract WildcardsQV is Initializable {
     ////////////////////////////////////
     //////// SETUP CONTRACT////////////
     //// NOTE: Upgradable at the moment
-    function initialize(uint256 _votingInterval) public initializer {
+    function initialize(uint256 _votingInterval, ILoyaltyToken _addressOfLoyalyTokenContract, IWildCardToken _addressOfWildCardTokenContract, IWildCardSteward _addressOfWildCardStewardContract, address _addressOfDragonCard) public initializer {
         admin = msg.sender;
         votingInterval = _votingInterval;
 
         proposalDeadline = now.add(_votingInterval);
+
+        loyaltyToken = _addressOfLoyalyTokenContract;
+        wildCardToken = _addressOfWildCardTokenContract;
+        wildCardSteward = _addressOfWildCardStewardContract;
+        addressOfDragonCard = _addressOfDragonCard;
```

Comment:
Won't need to pass this in :) We'll fetch the address of the owner of the dragon card everytime the increaseIteration is increased :)

Reply:
I am passing in the ID instead.